### PR TITLE
doc: Added VSC to bootloader sample doc

### PR DESCRIPTION
**Pull request by FrancescoSer**
Added VSC to bootloader sample doc.

Signed-off-by: Francesco Domenico Servidio <francesco.servidio@nordicsemi.no>

### CODE REVIEW

**Review by hakonfam — APPROVED** [review]
Nice!

File: samples/bootloader/README.rst [hunk 1]
```diff
@@ -136,9 +136,18 @@ To perform a full erase, do the following:
...
 
-   .. tab:: |SES|
+   .. tab:: |VSC|
 
-      Choose :guilabel:`Target` -> :guilabel:`Connect J-Link` and then :guilabel:`Target` -> :guilabel:`Erase All`.
+      Using the :guilabel:`ACTIONS` panel:
+
+         1. Go to the :guilabel:`ACTIONS` panel.
+         #. Move the cursor over the :guilabel:`Flash` action.
+         #. Click the :guilabel:`Erase and Flash to Board` button on the right side of the :guilabel:`Flash` action
```
Comment (b-gent) [inline samples/bootloader/README.rst:145]:
```suggestion
         #. Click :guilabel:`Erase and Flash to Board` on the right side of the :guilabel:`Flash` action.
```

File: samples/bootloader/README.rst [hunk 2]
```diff
@@ -247,35 +256,35 @@ To reduce the development time and potential issues with this route, let the exi
...
-   #. Click :guilabel:`Configure`.
-
-#. Select :guilabel:`Build` -> :guilabel:`Build Solution` to compile your application.
+   #. Click :guilabel:`Save`.
 
+#. Select :guilabel:`Build` in the :guilabel:`ACTIONS` panel to start the build process.
    The build process creates two images, one for the bootloader and one for the application, and merges them.
 
-#. Select :guilabel:`Build` -> :guilabel:`Build and Run` to program the resulting image to your device.
+#. Select :guilabel:`Flash` in the :guilabel:`ACTIONS` panel to program the resulting image to your device.
```
Comment (greg-fer) [inline samples/bootloader/README.rst:287]: Maybe you could use one of the images from Zigbee QSG? See the image under step 4 in the Step 2 section. https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_zigbee_qsg.html

**Review by greg-fer — APPROVED** [review]

File: samples/bootloader/README.rst [hunk 3]
```diff
@@ -247,35 +256,35 @@ To reduce the development time and potential issues with this route, let the exi
...
 1. :ref:`Create a private key in PEM format <ug_fw_update_keys>`.
-#. To enable the |NSIB|, run ``menuconfig`` on your application:
+#. Enable the |NSIB| through Kconfig as follows:
 
-   a. Select :guilabel:`Project` > :guilabel:`Configure nRF Connect SDK project`.
-   #. Go to :guilabel:`Modules` > :guilabel:`Nordic nRF Connect` > :guilabel:`Bootloader` and set :guilabel:`Use Secure Bootloader` to enable :kconfig:option:`CONFIG_SECURE_BOOT`.
-   #. Under :guilabel:`Private key PEM file` (:kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE`), enter the path to the private key that you created.
+   a. Select :guilabel:`Kconfig` in the :guilabel:`ACTIONS` panel to open the Kconfig configuration tab.
+   #. Expand :guilabel:`Modules` > :guilabel:`nrf` > :guilabel:`Nordic nRF Connect` > :guilabel:`Bootloader` and set :guilabel:`Use Secure Bootloader` to enable :kconfig:option:`CONFIG_SECURE_BOOT`.
+   #. Expand :guilabel:`Use Secure Bootloader` and enter, under :guilabel:`Private key PEM file` (:kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE`), the path to the private key that you created.
```
Comment (b-gent) [inline samples/bootloader/README.rst:271]:
```suggestion
   #. Expand :guilabel:`Use Secure Bootloader`.
      Under :guilabel:`Private key PEM file` (:kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE`), enter the path to the private key that you created.
```

File: samples/bootloader/README.rst (hunk 2 above)
Comment (FrancescoSer) [reply samples/bootloader/README.rst:287]: @greg-fer Are they even following the new style guide for screenshots?

**Review by b-gent — CHANGES_REQUESTED** [review]

File: samples/bootloader/README.rst (hunk 1 above)
Comment (FrancescoSer) [reply samples/bootloader/README.rst]: fixed

File: samples/bootloader/README.rst (hunk 3 above)
Comment (FrancescoSer) [reply samples/bootloader/README.rst]: fixed

**Review by b-gent — APPROVED** [review]

**Pull request merged**